ecs-cli/modules/commands/compose/compose_command.go

```diff
@@ -96,6 +100,10 @@ func composeFlags() []cli.Flag {
 			Name:  command.TaskRoleArnFlag,
 			Usage: "[Optional] Specifies the short name or full Amazon Resource Name (ARN) of the IAM role that containers in this task can assume. All containers in this task are granted the permissions that are specified in this role.",
 		},
+		cli.StringFlag{
+			Name:  command.ECSParamsFileNameFlag,
+			Usage: "[Optional] Specifies ecs-params file to use. Defaults to " + ecsParamsFileNameDefaultValue + " file, if one exists.",
+		},
```

Comment:
If you have `Value: ecsPramsFileNameDefaultValue` it will actually default to the value and also be added to the help text.

Reply:
If this is done, it makes error checking more difficult though. The default value of the file will always be there, even though the file is optional. For this reason I personally recommend against setting default values in the flag for optional fields.

Reply:
For example, if the user did not mean to use the optional ECS params file and does not specify it as an argument, the default file name will be in the value of the flag. But the file won't be there so if you throw a file not found error then you'd be breaking the CLI for people who don't use this file (and if you don't throw an error then you wouldn't give useful errors to people who do use the file).